View/src/view.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * view.h
+ *
+ *  Created on: Feb 15, 2015
+ *      Author: edem
+ */
+
+#ifndef VIEW_H_
+#define VIEW_H_
+
+#include "observer.h"
+
+/* This abstract class inherits observer's
+ * functionality and displays the data via use of
+ * the qt framework. In addition it receives
+ * commands from the user and sends them to the
+ * controller to be implemented
+ */
+
+class view : public observer
+{
+public:
+		view();
+
```

Comment:
I feel like these might be getting too far down into the Controller side of things.  View should focus on display and input gathering.  So information sent to Controller should be things like, mouse was clicked at position (x, y) or this button was clicked.  These functions look a bit more like controller commands.  I might be misunderstanding though.  Could you please clarify?